src/components/landing-components/trending.jsx

```diff
@@ -1,57 +1,23 @@
-import trendImg1 from "assets/images/book-covers/savarkar.jpg";
-import trendImg2 from "assets/images/book-covers/one-arr-mur.jpg";
-import trendImg3 from "assets/images/book-covers/auto-yogi.jpg";
-import trendImg4 from "assets/images/book-covers/chanakya-neeti.jpg";
-import trendImg5 from "assets/images/book-covers/yolo.jpg";
-import trendImg6 from "assets/images/book-covers/savarkar2.jpg";
+import { useState, useEffect } from "react";
+import axios from "axios";
 import { TrendingCard } from "components/cards/cards";
 
 export default function Trending() {
+  const [products, setProducts] = useState([]);
+  useEffect(() => {
+    (async () => {
+      const response = await axios.get("/api/products");
+      setProducts(response.data.products.slice(0, 6));
+    })();
+  }, []);
+
   return (
     <>
       <h2 className="h--2 trending__heading m__t-1 m__b-0">Trending</h2>
       <section className="trending flex justify--center p--x-2 p--y-2">
-        <TrendingCard
-          trendImg={trendImg1}
```

Comment:
Why did you passing data like this

Reply:
Bro you those lines are in removed, see the minus on left?